integration-tests/integration_test.go

```diff
@@ -166,13 +168,13 @@ func TestConnectionsAndEndpointsNormal(t *testing.T) {
 		Server: suites.Container{
 			Name: "socat-server-0",
 			Cmd:  "socat TCP4-LISTEN:40,reuseaddr,fork - &",
-			ExpectedNetwork: []types.NetworkInfo{
+			ExpectedNetwork: []*sensorAPI.NetworkConnection{
 				{
-					LocalAddress:   ":40",
-					RemoteAddress:  "CLIENT_IP",
-					Role:           "ROLE_SERVER",
-					SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
-					CloseTimestamp: types.NilTimestamp,
+					LocalAddress:   types.CreateNetworkAddress("", "", 40),
```

Comment:
I think it's worth it here and in other similar cases to introduce a separate variable keeping the client port. This will explain why there is 40 here and few lines above.

Reply:
Done